src/receipt.rs

```diff
@@ -1,4 +1,5 @@
 use alloc::vec::Vec;
+use core::fmt::Debug;
```

Comment:
Also unnecessary, the proc-macro for debug is included in the libcore v1 prelude:

https://doc.rust-lang.org/core/prelude/v1/index.html
> pub use crate::fmt::macros::[Debug](https://doc.rust-lang.org/core/fmt/macro.Debug.html);